### PR TITLE
Acknowledge JACK

### PR DESCRIPTION
**Pull request by pljones**
**Short description of changes**

Add a line to `README.md` acknowledging JACK (as it was the first audio interface library in use in Jamulus, it only seems fair).

CHANGELOG: SKIP

**Context: Fixes an issue?**

n/a

**Does this change need documentation? What needs to be documented and how?**

n/a

**Status of this Pull Request**

See https://github.com/pljones/jamulus/tree/add-jack-ack

**What is missing until this pull request can be merged?**

Needs review and comments.

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

**Review by ann0see — APPROVED** [review]
Thanks

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ to thank the maintainers of these projects for making their efforts available to
   [ccrma.stanford.edu/software/stk/](https://ccrma.stanford.edu/software/stk/)
 *   Some pixmaps are from the Open Clip Art Library (OCAL): [openclipart.com](https://openclipart.org/)
 *   Country flag icons from Mark James: [famfamfam.com](http://www.famfamfam.com)
+*   [JACK Audio Conection Kit](https://jackaudio.org/) by [the authors](https://github.com/jackaudio/jack2/blob/develop/AUTHORS.rst?plain=1)
```
Comment (ann0see) [inline README.md:50]: Even though not that important
```suggestion
*   [JACK Audio Conection Kit](https://jackaudio.org/) by [the JACK authors](https://github.com/jackaudio/jack2/blob/develop/AUTHORS.rst?plain=1)
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ to thank the maintainers of these projects for making their efforts available to
   [ccrma.stanford.edu/software/stk/](https://ccrma.stanford.edu/software/stk/)
 *   Some pixmaps are from the Open Clip Art Library (OCAL): [openclipart.com](https://openclipart.org/)
 *   Country flag icons from Mark James: [famfamfam.com](http://www.famfamfam.com)
+*   [JACK Audio Conection Kit](https://jackaudio.org/) by [the JACK authors](https://github.com/jackaudio/jack2/blob/develop/AUTHORS.rst?plain=1)
```
Comment (mcfnord) [inline README.md:50]:
```suggestion
*   [JACK Audio Connection Kit](https://jackaudio.org/) by [the JACK authors](https://github.com/jackaudio/jack2/blob/develop/AUTHORS.rst?plain=1)
```

Comment (pljones) [reply README.md:50]: 🤦

**Review by ann0see — APPROVED** [review]

**Pull request merged**